DPGAnalysis/MuonSysAging/python/RPCChamberMasker_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+RPCChamberMasker = cms.EDProducer('RPCChamberMasker',
+                                  digiTag = cms.InputTag('preRPCDigis'),
+                                  descopeRE31 = cms.bool(False),
+                                  descopeRE41 =cms.bool(False)
+				  )
+
+from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
+
+if phase2_muon.isChosen():
```

Comment:
toModify() won't make the job (easily), the two customise actually append "aged digis into different sequences" (RawToDigi vs localreco) because RPC digis skip the packing/unpaking in phase-2 as there are RE3/1 and RE4/1.
Therefore the whole process needs to be customised differently.
If you agree I will try using `makeProcessModifier()`, but it implies some more gymnastic.